utils/medium.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import json
+import logging
+from datetime import datetime
+
+
+def update_articles(db_conn, medium_url):
+    c = db_conn.cursor()
+    c.execute("DELETE FROM blogs")  # Clear all entries
```

Comment:
Clears manually added bogs each time the server starts. This is not desired. Try persisting the previous entries.

Reply:
I have updated as follow:

- Add new column `automatically_added` type INTEGER to table `blogs` to indicate if an entry is automatically added. 1 indicates True, 0 and NULL indicate False.
- When the function of updating blogs is triggered, it first deletes all the automatically added entries with `DELETE FROM blogs WHERE automatically_added = 1`, and then add crawled articles from Medium. So, manually added entries are kept.
- Update the code adding this column if missing when the app starts (@ db.py)

Reply:
The lasted commit is [here](https://github.com/Naresh1318/Topaz/pull/4/commits/5faa7fc17a40597f10733e4ebf61fe62268e32e5).